etl-job/fhir_import_export.py

```diff
@@ -199,8 +202,11 @@ def _load_all(program: str,
         db_path = (work_path / "local_fhir.db")
         db_path.unlink(missing_ok=True)
 
+        print("loading sqlite db...")
+        output["logs"].append("loading sqlite db...")
+
         db = LocalFHIRDatabase(db_name=db_path)
-        db.bulk_insert_data(resources=get_project_data(_get_grip_service(), "CALIPER", f"{program}-{project}", output, _get_token()))
+        db.bulk_insert_data(resources=get_project_data(hostname, "CALIPER", f"{program}-{project}", output, _get_token(), 1024*1024))
```

Comment:
Change graph name to "CALYPR"?

Reply:
Ok this should be addressed in recent commit here and in gen3-helm